public/mockServiceWorker.js

```diff
@@ -1,11 +1,14 @@
+/* eslint-disable */
```

Comment:
this is inserted automatically right? if not can we remove it?

Reply:
let me try

Reply:
I think they are automatically generated, I tried to remove them but then they appear again, how does @0xmad think?